package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Higher-order functions and common patterns for asynchronous code",
   "main": "lib/async.js",
   "author": "Caolan McMahon",
-  "version": "1.3.0",
+  "version": "1.2.1",
```

Comment:
whoops, didn't mean to commit that (it was just for benchmarking until #763 is figured out)